plugins/wallabag.koplugin/main.lua

```diff
@@ -41,71 +55,120 @@ local Wallabag = WidgetContainer:extend{
     name = "wallabag",
 }
 
+--- Add wallabag events to event dispatcher.
```

Comment:
Nit, but please refrain from adding useless comments that paraphrase the function name.

Reply:
I figured those would help complete <https://koreader.rocks/doc/modules/koplugin.wallabag.html>, but no worries, I've removed any superfluous ones